clients/curl-client.rst

```diff
@@ -12,17 +12,16 @@ To install the cURL client, run:
 
     $ composer require php-http/curl-client
 
-Usage
------
-
-The cURL client needs a :ref:`message <message-factory>` and a :ref:`stream <message-factory>`
-factory in order to to work::
+.. include:: includes/install-message-factory.inc
 
-    use Http\Client\Curl\Client;
+.. include:: includes/install-discovery.inc
```

Comment:
>  +The cURL client needs a :ref:`message factory <message-factory>` and

I meant this. If the message factory is necessary, it probably worths mentioning in the install instructions:

http://docs.php-http.org/en/latest/clients/guzzle5-adapter.html#installation

There are two include files for that.